### PR TITLE
Feat/testing continued

### LINKED ISSUE #14: Testing: Error Component
Need a test for the following user story:

- [x] As a user, when I visit the home, favorites or outfits page and the server's response is not okay I should see an Error component.

### LINKED ISSUE #15: Testing: Loading Component
Need a test for the following user story:

- [x] As a user, when I visit the homepage and the products are still being rendered I should see a Loading component.

### LINKED ISSUE #16: Testing: No Favorites div
Need a test for the following user story:

- [x] As a user, when I visit the favorites page and I haven't favorited any products I should see a div that tells me to "Add favorites to see them here.

### LINKED ISSUE #17: Testing: Home route
Need a test for the following user story:

- [x] As a user, when I visit the homepage the URL should be "/".

- [x] As a user, when I click on the aside's "All Items" link, I should be directed to the homepage.

### LINKED ISSUE #18: Testing: Favorite Route
Need a test for the following user story:

- [x] As a user, when I visit the favorites page the URL should be "/favorites".

- [x] As a user, when I visit the favorites page I should only see items that I have favorited.

### PR DESCRIPTION
**Pull request by Markus-Xavier**
What (if any) features are you implementing?
- Add testing for routes, favorites page, errors and loading 
What (if anything) did you refactor?
- N/A
Were there any issues that arose?
- N/A
Is there anything that you need from your teammate?
- N/A
Any other comments, questions, or concerns?
Review and suggest changes!

### CODE REVIEW

**Comment by ivonne-hernandez** [comment]
Markus, can you please add a test for when a user clicks on the aside's "All Items" link to ensure that it navigates them to the homepage?

**Comment by Markus-Xavier** [comment]
I have added the discussed tests to this branch, requesting review @ivonne-hernandez @isleofyou

File: cypress/fixtures/productsNoFav.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  },
+  {
+    "name": "Full Day Ahead Short Sleeve",
+    "color": "Aquila Rhino Grey Multi",
+    "img_url": "https://images.lululemon.com/is/image/lululemon/LW3EMPS_049466_1?wid=1080&op_usm=0.5,2,10,0&fmt=webp&qlt=80,1&fit=constrain,0&op_sharpen=0&resMode=sharp2&iccEmbed=0&printRes=72",
+    "category": "top",
+    "id": 3,
+    "favorite": false
+  }
+]
```
Comment (ivonne-hernandez) [inline cypress/fixtures/productsNoFav.json:26]: Nice! It was a really good idea to create another fixture for this "no favorites" test.

**Review by ivonne-hernandez — APPROVED** [review]
Thank you so much for taking on these tests Markus! This branch is ready to be merged.

**Pull request merged**